### DISCUSSION TITLE
Domain Prefetching too agressive

### DISCUSSION THREAD

**Discussion by c-michail** [mokeyish/smartdns-rs#370] [General]
Hello,
  I enabled domain prefetching and it's too aggressive. According to my DNS server it does around 2000 queries per minute.
  Does it ever stop ? (atm i disabled it)
  
btw 
you should disable the dns prefetch thread in case it's disabled in the config.
(DEBUG:smartdns::dns_mw_cache:315: Domain prefetch check will be performed in 6.3393002s.)

Thanks

**Comment by mokeyish** [comment]
Currently, you can set enviroment varianle `PREFETCH_MIN_INTERVAL` to adjust its frequence.

https://github.com/mokeyish/smartdns-rs/blob/188fca40f4d22506142c33617e05811e7bfd57db/src/dns_mw_cache.rs#L96

**Reply by c-michail** [reply]
It would be nice to add is as a parameter to the config file.

Thank you

**Reply by c-michail** [reply]
Hello,
  It doesn't seem to have any effect on my linux vm. (It works on windows though)
![εικόνα](https://github.com/user-attachments/assets/13cc630f-0c95-4162-a9fd-7b722739ef1a)

  Can you please make it as a parameter in the config file ?

Thank you

**Reply by c-michail** [reply]
@mokeyish not improved in v0.8.8-alpha.1
